src/fastmath/stats.clj

```diff
@@ -273,17 +273,17 @@
   ([vs p1 p2] (percentile-bc-extent vs p1 p2 :legacy))
   ([vs ^double p1 ^double p2 estimation-strategy]
    (let [avs (m/seq->double-array vs)
-         gaussian (r/distribution :normal {:mu 0 :sd 1})
+         m (mean avs)
         ;;  icdf of the number of bootstrap samples <= the mean
-         z0 (r/icdf gaussian (/ (double (count (filter  #(<= % (mean avs)) vs))) (count vs)))
-         z1 (r/icdf gaussian (/  (double p1) 100))
-         z2 (r/icdf gaussian (/  (double  p2) 100))
-         q1 (r/cdf gaussian (+ (* 2 z0) z1))
-         q2 (r/cdf gaussian (+ (* 2 z0) z2))]
+         z0 (r/icdf r/default-normal (/ (count (filter  #(<= % m) vs)) (count vs)))
```

Comment:
`z0`, `z1` and `z2` should be type hinted.

```clojure
^double z0 (r/icdf ...)
```

Filter predicate also should contain type hint, so instead of:

```clojure
#(<= % m)
```

You can make:

```clojure
#(<= ^double % m)
;; or
(fn [^double v] (<= v m))
```

You removed `double` here but it's necessary to avoid integer division. Fastmath overrites `/` so it do not produce `ratio` anymore:

```clojure
(/ (double (count ...)) (count vs))
```